sources/academy/webscraping/anti_scraping/index.md

```diff
@@ -93,7 +93,9 @@ A common workflow of a website after it has detected a bot goes as follows:
 
 One thing to keep in mind while navigating through this course is that advanced scraping methods are able to identify non-humans not only by one value (such as a single header value, or IP address), but are able to identify them through more complex things such as header combinations.
 
-> For an in-depth video explanation of anti-scraping, and how to fix issues caused by it, check out [this talk](https://www.youtube.com/watch?v=aXil0K-M-Vs) from [Ondra Urban](https://github.com/mnmkng).
+A conference talk by [Ondra Urban](https://github.com/mnmkng) will guide you through various anti-scraping measures and how to get around them.
```

Comment:
@honzajavorek Some of the parts of the video are outdated. Especially the code examples. What's your take on it? Should we explicitly mention that? Or do people understand that videos get outdated and don't care that the code examples are no longer useful in verbatim?

Reply:
Good point. I think it could be accommodated by the text attached. We could mention it's 2021 and warn that something might be off, but for a general idea it should be okay.